test/built-ins/System/System.js

```diff
@@ -0,0 +1,30 @@
+// Copyright (C) 2016 Jordan Harband. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: system
```

Comment:
I believe this should be `esid: sec-system` or if the id is not defined yet we can use `esid: pending`.